src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    application from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest upgrades to unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about new applications</string>
+  <string name="update_notifications_summary">Show a notification when new versions are available</string>
   <string name="updates">Updates</string>
   <string name="updates_empty_summary">All applications up to date</string>
   <string name="username">Username</string>
-  <string name="username_is_not_specified">Username is not specified</string>
-  <string name="validation_index_error_description">Failed to validate the index.</string>
-  <string name="validation_package_error_description_format">Failed to validate the package: %s.</string>
-  <string name="validation_error_hash_lower">invalid hash</string>
-  <string name="validation_error_integrity_lower">integrity check failed</string>
-  <string name="validation_error_permissions_lower">missing permissions</string>
-  <string name="validation_error_signature_lower">invalid signature</string>
-  <string name="validation_error_structure_lower">invalid file format</string>
+  <string name="username_is_not_specified">Username not specified</string>
+  <string name="validation_index_error_description">Could not validate the index.</string>
+  <string name="validation_package_error_description_format">Could not validate the package: %s.</string>
+  <string name="validation_error_hash_lower">Invalid checksum</string>
+  <string name="validation_error_integrity_lower">Could not check integrity</string>
+  <string name="validation_error_permissions_lower">Add reqiored permissions</string>
+  <string name="validation_error_signature_lower">Invalid signature</string>
+  <string name="validation_error_structure_lower">Invalid file format</string>
   <string name="version_format">Version %s</string>
   <string name="waiting_to_start_download">Waiting to start download</string>
   <string name="website">Website</string>
-  <string name="whats_new">What\'s new</string>
+  <string name="whats_new">News</string>
```

Comment:
"What's new" is better here because this block often contains the list of changes.

Reply:
It routinely is out of bounds though, even on older F-Droid. Why is a list of changes not "News"?
In my head a historic list of news being news isn't much of an abstraction.

Reply:
I'll probably rename it to "Latest changes" or "Change log", this looks even better to me.

Reply:
"Latest" or "Changes" looks good. "Changelog" avoids confusion.

Reply:
"News" and "application news" ties things together, but I am unsure whether it will work.
Reverted